packages/backend/server/src/__tests__/copilot.spec.ts

```
@@ -1931,3 +1935,71 @@ test('should handle generateSessionTitle correctly under various conditions', as
+    await cronJobs.dailyCleanupJob();
+    t.snapshot(
+      jobAddStub.getCalls().map(call => ({
+        args: call.args,
+      })),
+      'daily job scheduling calls'
+    );
+
+    jobAddStub.reset();
+    cleanupStub.reset();
+    toBeGenerateStub.reset();
+  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)